drivers/SmartThings/matter-thermostat/profiles/fan-generic.yml

```diff
@@ -0,0 +1,14 @@
+name: fan-generic
```

Comment:
I think we should switch the generic fingerprint to this profile since not all fans will support wind

Reply:
agreed

Reply:
though with this dynamic profiling setup, any profile would be updated anyway. Still a good addition for clarity